sqlglot/dialects/tsql.py

```diff
@@ -630,7 +631,18 @@ class Generator(generator.Generator):
             exp.VolatileProperty: exp.Properties.Location.UNSUPPORTED,
         }
 
-        LIMIT_FETCH = "FETCH"
+        def boolean_sql(self, expression: exp.Boolean) -> str:
+            bit_types = {exp.EQ, exp.NEQ, exp.Is, exp.In, exp.Select, exp.Alias}
```

Comment:
Let's factor this out of T-SQL's class altogether so it's not redefined in each call, i.e. at the top-level of `tsql.py`. Let's also name it `BIT_TYPES` to be consistent with constant naming.